src/main/java/com/paneedah/mwc/tileentities/TileEntities.java

```diff
@@ -1228,7 +1250,7 @@ public static void init(CommonProxy commonProxy) {
         new LootBoxConfiguration()
         .withMaterial(Material.IRON)
         .withName("ServerRackLeftAlt3")
-        .withModelClassName("com.paneedah.mwc.models.ServerRackLeftAlt3")
+        .withModelClassName("com.paneedah.mwc.models.ServerRackLeftAlt2")
```

Comment:
Why? It is named `Alt3` so why is it using `Alt2`

Reply:
There is no icon for Alt3 so to make it different than the sandbag and stuff I made it use the Alt2 texture

Reply:
Yeah but `withModelClassName` will make it use the model of Alt2

Reply:
No, <code>.withTextureName</code> is the model (Dont ask me why idk)

I changed it when I fixed the office chairs (#193)

Reply:
Definitly not `.withTextureName` points towards the png of the texture as it should, and looked inside the code it is the model

Reply:
> No, `.withTextureName` is the model (Dont ask me why idk)
> 
> I changed it when I fixed the office chairs (#193)

Oh crums, well yeah the icons are tied to the model

Reply:
idk at this point. I'll change it back

Reply:
Fixed